### PR TITLE
[JAVA] fixed white line in javadoc comment java pojo issue that breaks maven…

### PR DESCRIPTION
**Pull request by gurubook**
fix pojo template for javadoc generation

### Description of the PR
small fix to allow maven build of generated java pojo

### CODE REVIEW

**Comment by wing328** [comment]
Thanks for the PR but your commit (as shown in the Commits tab) is not linked to your Github account, which means this PR won't count as your contribution in https://github.com/swagger-api/swagger-codegen/graphs/contributors.

Let me know if you need help fixing it.

Ref: https://github.com/swagger-api/swagger-codegen/wiki/FAQ#how-can-i-update-commits-that-are-not-linked-to-my-github-account

**Comment by gurubook** [comment]
Never mind, i'll fix for future contribution, go ahead.

Thank you
Luca

File: modules/swagger-codegen/src/main/resources/Java/pojo.mustache [hunk 1]
```diff
@@ -50,22 +50,22 @@ public class {{classname}} {{#parent}}extends {{{parent}}} {{/parent}}{{#parcela
...
-   * Get {{name}}
-  {{/description}}
-  {{#minimum}}
-   * minimum: {{minimum}}
-  {{/minimum}}
-  {{#maximum}}
-   * maximum: {{maximum}}
-  {{/maximum}}
-   * @return {{name}}
+
```
Comment (wing328) [inline modules/swagger-codegen/src/main/resources/Java/pojo.mustache]: @gurubook There are 2 white spaces in this line. Are these required?

**Comment by gurubook** [comment]
Double space at line 54 can be removed.
Thanks
Luca

**Comment by wing328** [comment]
@gurubook I didn't get any error when running `mvn javadoc:javadoc`:
```
Generating /Users/williamcheng/Code/jan2017/swagger-codegen/samples/client/petstore/java/okhttp-gson/target/site/apidocs/allclasses-frame.html...
Generating /Users/williamcheng/Code/jan2017/swagger-codegen/samples/client/petstore/java/okhttp-gson/target/site/apidocs/allclasses-noframe.html...
Generating /Users/williamcheng/Code/jan2017/swagger-codegen/samples/client/petstore/java/okhttp-gson/target/site/apidocs/index.html...
Generating /Users/williamcheng/Code/jan2017/swagger-codegen/samples/client/petstore/java/okhttp-gson/target/site/apidocs/overview-summary.html...
Generating /Users/williamcheng/Code/jan2017/swagger-codegen/samples/client/petstore/java/okhttp-gson/target/site/apidocs/help-doc.html...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 5.942 s
[INFO] Finished at: 2017-02-09T01:14:51+08:00
[INFO] Final Memory: 15M/309M
[INFO] ------------------------------------------------------------------------
okhttp-gson|master⚡ ⇒ mvn javadoc:javadoc
```

Which version of Swagger Codegen are you using?

**Comment by gurubook** [comment]
Will break if a schema object in a definition section have something in a property description field, and petstore "sample.json" does not have any.
adding something like this might cause the issue 
`
    "User": {
        "properties": {
          "id": {
           "description" : "the user id",
            "type": "integer",
            "format": "int32"
          },
          "lastName": {
           "description" : "the user lastName",
            "type": "string"
          },
          "username": {
           "description" : "the user name",
            "type": "string"
          },
          "phone": {
            "type": "string"
          },
          "email": {
            "type": "string"
          },
          "userStatus": {
            "type": "integer",
            "format": "int32"
          },
          "firstName": {
            "type": "string"
          },
          "password": {
            "type": "string"
          }
        }
      },`

**Comment by wing328** [comment]
@gurubook OK. I'll try to reproduce the issue.

**Comment by wing328** [comment]
https://github.com/swagger-api/swagger-codegen/blob/master/modules/swagger-codegen/src/test/resources/2_0/petstore-with-fake-endpoints-models-for-testing.yaml#L902) for testing, which contains properties with description. 

Can you share the spec with me so that I can use it to reproduce the issue?

(I assume you encountered the issue with the latest master, not stable releases)

**Comment by wing328** [comment]
Can you also try running `mvn javadoc:javadoc` under `samples/client/petstore/java/okhttp-gson`?

**Comment by gurubook** [comment]
You're right, everything works as expected, i tracked down the issue on my side.
I was still use an outdated jmustache 1.9 that incorrectly handle test wrapped in javadoc comment in my project.
That would override the 1.12 version in codegen project, and cause that issue.

Sorry for the wrong report, and thank you for your support.
Luca

**Comment by wing328** [comment]
@gurubook no problem and we're glad that you filed a PR to help improve Swagger Codegen.

Looking forward to more contributions from you.

**Pull request closed without merging**